tla-bmcmt/src/test/scala/at/forsyte/apalache/tla/bmcmt/stratifiedRules/aux/oracles/TestIntOracle.scala

```diff
@@ -126,7 +126,8 @@ class TestIntOracle extends AnyFunSuite with BeforeAndAfterEach with Checkers {
   }
 
   // We cannot test getIndexOfChosenValueFromModel without running the solver
-  test("getIndexOfChosenValueFromModel recovers the index correctly") {
+  // Ignored until we figure out why it's killing GH CLI
+  ignore("getIndexOfChosenValueFromModel recovers the index correctly") {
```

Comment:
Can we
- [ ] refer to the relevant issue here, and
- [ ] leave a note under that issue to (re-)enable these tests?